git/gogit/clone_test.go

```diff
@@ -896,6 +898,189 @@ func Test_getRemoteHEAD(t *testing.T) {
 	g.Expect(head).To(Equal(fmt.Sprintf("%s/%s", "v0.1.0", cc)))
 }
 
+func TestClone_CredentialsOverHttp(t *testing.T) {
+	tests := []struct {
+		name                     string
+		username                 string
+		password                 string
+		allowCredentialsOverHttp bool
+		transformURL             func(string) string
+		expectNewErr             string
```

Comment:
This field is not specified in the any of the test cases below.

Reply:
That was due to change where the validation was occurring, this is now fixed in both libgit2 and gogit.